embassy-usb/src/lib.rs

```diff
@@ -376,6 +370,24 @@ impl<'d, D: Driver<'d>> Inner<'d, D> {
                     h.suspended(true);
                 }
             }
+            Event::PowerDetected => {
+                trace!("usb: power detected");
+                self.bus.enable().await;
```

Comment:
The driver should do this itself inside the `poll` function

Reply:
My understanding is that this is no longer an issue given the agreement on Matrix that we'll use my approach.